docs/api/python/symbol.md

```diff
@@ -392,6 +392,21 @@ Composite multiple symbols into a new one by an operator.
     argmin
 ```
 
+### Linear Algebra
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    la_gemm
```

Comment:
Yes, Zhenwen is already working on this. We want to get it done  soon as we want to provide a truly full and scalable solution. Though I would prefer to get the CPU-versions in first and the GPU ones with a separate PR. Even if this means that we have to do a bit of restructuring of code afterwards again (in order to unify the cpu/gpu abstraction). 
I will rename the stuff to linalg.